internal/flag/flags.go

```diff
@@ -42,6 +42,7 @@ const (
 	AuthorizationEnabled            = "authorizationEnabled"            // AuthorizationEnabled flag
 	Provider                        = "provider"                        // Provider flag to set the cloud provider
 	Region                          = "region"                          // Region flag
+	Regions                         = "regions"                         // Regions flag
```

Comment:
[nit] we keep options singular even if they allow multiple values, reason being this is also valid
```bash
mongocli --region a --region b
```